spec/dummy/db/migrate/20230627061428_create_users.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
I saw recommendations of https://github.com/pat/combustion for engines testing. Maybe it'll make some things easier

Reply:
Added task to try this